docs/deploy-rbd.md

```diff
@@ -196,6 +198,28 @@ The Helm chart is located in `charts/ceph-csi-rbd`.
 
 [See the Helm chart readme for installation instructions.](../charts/ceph-csi-rbd/README.md)
 
+## Read Affinity using crush locations for RBD volumes
+
+Ceph CSI supports mapping RBD volumes with krbd options
+`"read_from_replica=localize,crush_location=type1:value1|type2:value2"` to
+optimize for reads from the most local serving OSD (OSD assigned in the
+same ceph CRUSH map location i.e.,region,zone,rack,etc).
+Refer [krbd-options](https://docs.ceph.com/en/latest/man/8/rbd/#kernel-rbd-krbd-options)
+for more details.
+
+This can be enabled by adding labels to Kubernetes nodes in the format `"topology.io/zone=east"`
+and passing cmdline arguments `"--enable-read-affinity=true"` and
+`"--crush-location-labels=topology.io/zone,topology.io/region"`
+to Ceph CSI RBD daemonset pod "csi-rbdplugin" container, resulting in Ceph CSI adding
+`"--options read_from_replica=localize,crush_location=zone:east"` krbd options during
+rbd map operation.
```

Comment:
I'd extend the example to two CRUSH locations:
```suggestion
This can be enabled by adding labels to Kubernetes nodes like
`"topology.io/region=east"` and `"topology.io/zone=east-zone1"` and
passing cmdline arguments `"--enable-read-affinity=true"` and
`"--crush-location-labels=topology.io/zone,topology.io/region"` to Ceph CSI
RBD daemonset pod "csi-rbdplugin" container, resulting in Ceph CSI adding
`"--options read_from_replica=localize,crush_location=zone:east-zone1|region:east"`
krbd options during rbd map operation.
```